src/tls.c

```diff
@@ -9008,7 +9008,7 @@ int TLSX_Parse(WOLFSSL* ssl, byte* input, word16 length, byte msgType,
             InitSSL_Method(method, MakeTLSv1_1());
     #endif
 #endif
-#ifndef NO_OLD_TLS
+#if !defined(NO_OLD_TLS) || defined(WOLFSSL_TLS13)
```

Comment:
Commit message says NO_OLD_TLS "IS" defined

This line of code handles NO_OLD_TLS "NOT" defined.  Those are opposite.

More importantly, users doing mostly TLS 1.2 but some TLS 1.3 who are using v23 to allow both and using NO_OLD_TLS to prevent TLS 1.1 and TLS 1.0 will suddenly have their code allowing TLS 1.1 and TLS 1.0.  That is a problem.

Reply:
if NO_OLD_TLS is defined then the TLS 1.1 and 1.0 code is not able to work and not compiled.
The change only allows downgrading from 1.3 to 1.2 when TLS 1.3 is built.

Reply:
Changing WOLFSSL_MIN_DOWNGRADE to TLS v1.2 when NO_OLD_TLS defined.
Fixed commit message.